add information about user feedback notifications (#9508)
getsentry/sentry-docs: docs/product/user-feedback/index.mdx

![User Feedback Alert Configuration.](./img/user-feedback-alert-configuration.png)

> To be alerted when crash-report feedback comes in, set up Workflow Notifications as described [here](https://docs.sentry.io/product/alerts/notifications/#workflow-notifications). Also, please make sure that Crash Report Notifications are enabled in Settings > Projects > [Project Name] > User Feedback page.

Reviewer: ```suggestion
To be alerted when crash-report feedback comes in, set up Workflow Notifications as described [here](https://docs.sentry.io/product/alerts/notifications/#workflow-notifications). Also, please make sure that Enable Crash Report Notifications are enabled in Settings > Projects > [Project Name] > User Feedback.
```

Reviewer 2: Modified your suggestion a bit

Reviewer 3: Lets remove the entire 'workflow notifications' sentence? We can simplify this 

Per https://sentry.slack.com/archives/C05J714QPRR/p1710967736899059?thread_ts=1710404908.615159&cid=C05J714QPRR

Reviewer 2: ```suggestion
To get notifications when crash-report feedback comes in, make sure to turn on "Enable Crash Report Notifications" in Settings > Projects > [Project Name] > User Feedback.
```
How's this?

Reviewer 3: Looks good!
